test/Algebra/Graph/Test/GraphRules.hs

```diff
@@ -0,0 +1,99 @@
+{-# LANGUAGE TemplateHaskell #-}
+-----------------------------------------------------------------------------
+-- |
+-- Module     : Algebra.Graph.Test.Graph
+-- Copyright  : (c) Andrey Mokhov 2016-2018
+-- License    : MIT (see the file LICENSE)
+-- Maintainer : andrey.mokhov@gmail.com
+-- Stability  : experimental
+--
+-- Testsuite for "Algebra.Graph" rewrite-rules
+-----------------------------------------------------------------------------
+module Algebra.Graph.Test.GraphRules where
```

Comment:
Let's call the module "RewriteRules"? I think it's a bit clearer.

Reply:
Done